pkgcfg/package-configuration/mariadb.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "mariadb",
+  "version": "0.0.1",
+  "env": {
+    "MYSQL_BASEDIR": "{{ .DevboxProfileDefault }}",
```

It might be a good idea to remove the quotes and then `json.Marshal` the string values:

```suggestion
    "MYSQL_BASEDIR": {{ .DevboxProfileDefault }},
```

That way characters get escaped properly.

---

These strings can be concatenated sometimes so I don't think to can do that. We could register a function so we could do:

`{{ escape .DevboxProfileDefault }}`

?